website/content/docs/agent/options.mdx

```diff
@@ -1770,6 +1770,15 @@ bind_addr = "{{ GetPrivateInterfaces | include \"network\" \"10.0.0.0/8\" | attr
 - `protocol` ((#protocol)) Equivalent to the [`-protocol` command-line
   flag](#_protocol).
 
+- `raft_boltdb` ((#raft_boltdb)) This is a nested object that allows configuring 
+  options for Rafts BoltDB based log store.
```

Comment:
Raft's

Reply:
and "BoltDB-based".  Two-word adjectives get hyphenated.  One of the few things that stuck with me from HS english.  :)